### ISSUE TITLE
Fix paths on VLC playlist entries

### ISSUE THREAD

**Issue by stblassitude** [stblassitude/obs-scene-transporter#1]
Paths of VLC playlist entries are not patched, and the entries are not recognized as assets.

Example partial json:
```
    {
        "id": "vlc_source",
        "settings": {
                "playback_behavior": "pause_unpause",
                "playlist": [
                    {
                        "hidden": false,
                        "selected": false,
                        "value": "/home/devday/Documents/Screens/172561__djgriffin__video-game-7.wav"
                    },
                    {
                        "hidden": false,
                        "selected": false,
                        "value": "/home/devday/Documents/Screens/240376__edtijo__happy-8bit-pixel-adenture.wav"
                    }
                ]
            },
    },
```

**Comment by stblassitude** [comment]
This might take a query tool for dicts and/or JSON to properly identify entries that are (or can be) filesystem paths, like https://github.com/akesterson/dpath-python

**Comment by stblassitude** [comment]
Fixed in 0.9.13
